extension.js

```diff
@@ -58,6 +65,22 @@ async function activate() {
 		}
 	});
 
+	// Event listener for when the window is focused
+	vscode.window.onDidChangeWindowState(async () => {
```

Comment:
@jlawrence124 I think that the idea is really really good, but I'm not totally convinced that the behaviour should be exactly like this. Example:
- Two projects in different windows
- Each project with a different default org
- `"settingsScope": "workspace"`
![image](https://github.com/victorgz/vscode-salesforce-colorg/assets/6944989/e0441c32-afb0-42cb-a786-011033720ba2)

Each time I focus on a windows, the color is disabled in the other(s) one(s) because the initial clean up.
I think that this logic should only apply for the users with `"settingsScope": "user"`, as the color config is stored at the user level.

What do you think?

Reply:
I was already working on something related to this so I think we're on the same page for sure.  This was also causing some issues on slower laptops (like my work one 😆 ) so I'll make sure i get it thoroughly tested on my end.

Reply:
I did a couple of changes to fix 🔧  this and also optimize for speed 🏃‍♂️ 💨 

🔧
- [Limited `initialCleanup` to only run if `"settingsScope": "user"` ](https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/545f0428fb3c0e4f1ef496ce0c36bda929b1e3c7)
-  [Made `vscode.window.onDidChangeWindowState` only run when settings scope is `user`](https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/dad18ba010984f98bf38ccd7abe2748502c6ec4a)

🏃‍♂️ 💨 
- [Bypassed `init` function when `vscode.window.onDidChangeWindowState` is called after initialization](https://github.com/victorgz/vscode-salesforce-colorg/pull/4/commits/18ba21c12bf0bd1969ee603e463d346a1790e9ba)